app/pods/components/tracks/tracks-doubts-form/component.js

```diff
@@ -18,6 +19,7 @@ export default class TracksDoubtsFormComponent extends Component {
           data: {
             name: this.name,
             mobile: this.mobile,
+            email: this.email,
```

Comment:
Is this working now? I don't think this will work unless we change the hubspot form